code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -930,8 +930,6 @@
 
 /datum/reagent/iron/expose_mob(mob/living/exposed_mob, methods=TOUCH, reac_volume)
 	. = ..()
-	if(!exposed_mob.has_bane(BANE_IRON)) //If the target is weak to cold iron, then poison them.
-		return
 	if(!holder || (holder.chem_temp >= 100)) // COLD iron.
 		return
 
```

Comment:
No, just remove this whole proc.  You made it so that taking a beaker full of iron, chilling it, and throwing it at anyone will poison them.